packages/trader/src/Stores/Modules/CFD/Helpers/cfd-config.ts

```diff
@@ -1,103 +1,106 @@
 import { localize } from '@deriv/translations';
 
+export type TDxCompanies = ReturnType<typeof getDxCompanies>;
+export type TMtCompanies = ReturnType<typeof getMtCompanies>;
+
 export const getDxCompanies = () => {
     const synthetic_config = {
         account_type: '',
         leverage: 500,
-        short_title: localize('Synthetic'),
+        short_title: localize('Synthetic') as string,
```

Comment:
there will be no need to explicitly assert that the return type of `localize()` is string after https://github.com/binary-com/deriv-app/blob/master/packages/translations/src/i18next/i18next.js is migrated to TS.